### PR TITLE
CIV-13537 LR vs Lip with notice

### PR DESCRIPTION
**Pull request by pliao-hmcts**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-13537

### Change description ###

LR vs Lip with notice

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json [hunk 1]
```diff
@@ -22,5 +22,19 @@
     "Label": "Is GA for LIP",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "claimantUserDetails",
+    "Label": " ",
+    "FieldType": "GAUserDetailsGAspec",
+    "SecurityClassification": "Public"
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json:31]: did we miss "Searchable": "N" as it's new field?

File: ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json [hunk 2]
```diff
@@ -22,5 +22,19 @@
...
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "defendantUserDetails",
+    "Label": " ",
+    "FieldType": "GAUserDetailsGAspec",
+    "SecurityClassification": "Public"
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json:38]: did we miss "Searchable": "N" as it's new field?

File: ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json (hunk 1 above)
Comment (pliao-hmcts) [reply ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json]: > did we miss "Searchable": "N" as it's new field?

added

File: ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json (hunk 2 above)
Comment (pliao-hmcts) [reply ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json]: > did we miss "Searchable": "N" as it's new field?

added

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Pull request merged**